src/core/i18n/utils.js

```diff
@@ -230,6 +232,56 @@ function oneLineTranslationString(translationKey) {
   return translationKey;
 }
 
+type FormatFilesizeParams = {|
+  _filesize: typeof filesize,
+  _log: typeof log,
+  i18n: I18nType,
+  size: number,
+|};
+
+// Translates a file size in bytes into a localized user-friendly format.
+export const formatFilesize = ({
+  _filesize = filesize,
+  _log = log,
+  i18n,
+  size,
+}: FormatFilesizeParams): string | null => {
+  const sizeStrings = {
+    // These are the expected values for the unit of measure returned by
+    // filesize. Realistically we shouldn't get anything back larger than TB.
+    // translators: B is an abbreviation of Bytes in English.
+    // Localize it if necessary but use a short abbreviation.
```

Comment:
It's a bit lame but you have to put the `translators:` prefixed comment all on one line. You can test it by running `bin/extract-locales` then checking `amo.pot`.

Reply:
Hmm, I was wondering about that so I checked some other code and saw translation notes on multiple lines, e.g., [here](https://github.com/mozilla/addons-frontend/blob/2018.10.11/src/amo/components/AddonMoreInfo/index.js#L125-L126) and [here](https://github.com/mozilla/addons-frontend/blob/2018.10.11/src/amo/pages/AddonReviewList/index.js#L302-L303). I guess those are bad. Maybe we need to open a code quality issue to check for multi-line translation hints?